cmd/bigsky/main.go

```diff
@@ -486,7 +490,8 @@ func runBigsky(cctx *cli.Context) error {
 	// set up metrics endpoint
 	go func() {
 		if err := bgs.StartMetrics(cctx.String("metrics-listen")); err != nil {
-			log.Fatalf("failed to start metrics endpoint: %s", err)
+			log.Error("failed to start metrics endpoint", "err", err)
+			os.Exit(1)
```

Comment:
any style reason to use `Exit(1)` vs `Exit(-1)`? I to the later by default but don't super care either way.

Reply:
I checked the source for stdlib log.Fatal and it's Exit(1)